### PR TITLE
Remove full tuple encrytion

### PR DESCRIPTION
**Pull request by dAdAbird**
Leaving tuple data (attributes) only encrytion

### CODE REVIEW

File: src/encryption/enc_tuple.c [hunk 1]
```diff
@@ -80,7 +80,7 @@ static void PGTdeDecryptTupInternal2(BlockNumber bn, Page page, HeapTuple tuple,
 	// Most of the time we can't decrypt in place, so we allocate some memory... and leek it for now :(
 	if(allocNew)
 	{
-		newPtr = malloc(tuple->t_len);
+		newPtr = palloc0(tuple->t_len);
```
Comment (dutow) [inline src/encryption/enc_tuple.c:83]: Unrelated change

Comment (dAdAbird) [reply src/encryption/enc_tuple.c]: fixed https://github.com/Percona-Lab/postgres-tde-ext/pull/4/commits/34ca182185b6c96dcdfa7c8c51f5709995d32f7b

File: src/encryption/enc_tuple.c [hunk 2]
```diff
@@ -138,37 +114,30 @@ PGTdePageAddItemExtended(Oid oid,
 
 	char* toAddr = ((char*)phdr) + phdr->pd_upper;
 
-#if FULL_TUPLE_ENCRYPTION
```
[the remark this answers is not in the document]
Comment (dAdAbird) [reply src/encryption/enc_tuple.c:141]: fixed https://github.com/Percona-Lab/postgres-tde-ext/pull/4/commits/34ca182185b6c96dcdfa7c8c51f5709995d32f7b

**Review by dutow — APPROVED** [review]
Looks good! Please use `squash and merge` so we won't end up with fixup comments in the main history :)
